control-plane/pkg/reconciler/broker/broker.go

```diff
@@ -274,15 +276,15 @@ func (r *Reconciler) reconcileBrokerTopic(broker *eventing.Broker, securityOptio
 
 func (r *Reconciler) FinalizeKind(ctx context.Context, broker *eventing.Broker) reconciler.Event {
 	return retry.RetryOnConflict(retry.DefaultBackoff, func() error {
-		return r.finalizeKind(ctx, broker)
+		return r.DoFinalizeKind(ctx, broker)
 	})
 }
 
-func (r *Reconciler) finalizeKind(ctx context.Context, broker *eventing.Broker) reconciler.Event {
+func (r *Reconciler) DoFinalizeKind(ctx context.Context, broker *eventing.Broker) reconciler.Event {
```

Comment:
I don't think the `RetryOnConfict` in the other reconciler is needed, we should just call `ReconciledKind` of this reconciler.

Reply:
LOL good point :)

Reply:
Done

Reply:
I think this is still a problem because we're still calling `Do*`

Reply:
Oh, nice catch. I messed up